src/uerouting.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
```

Comment:
What is this file used for?

And its content is quite different from the one in aether-in-a-box where there is only the following:

```
info:
  description: Routing information for UE
  version: 1.0.0
```

Reply:
Changed content as discussed.